README.md

```diff
@@ -1,42 +1,34 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Hello world Project
 
+> This project display hello word.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML5
+- CSS3
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
 ### Setup
 
-### Install
-
-### Usage
+#### clone this project to your local computer and use telminal togo into your the parent directory of this project.
```

Comment:
Your project looks perfect and ready we can make our project documentation even more professional. 😇
**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the correct project doesn't require testing.**

As mentioned in the template let's make sure we remove all sections that doesn't apply for example this comment where it gives you instructions of how to use the Microverse template.

1. Authors should be only one author and fill It with your social links for example LinkedIn/GitHub are good to have others I would not recommend unless they are professional.
2. Kind remainder let's make sure we also get rid of any text that doesn't apply.

Reply:
Thank you!, I have tried my best to work on feedback